docs/user-guide/lns-module-configuration.md

```diff
@@ -0,0 +1,38 @@
+# Network Server IoT Edge Module configuration
+
+The following table is providing a list of configuration options, to be provided as environment variables for manual configuration of the `LoRaWanNetworkSrvModule`:
+
+| Environment variable name   | Description                                                                            | Mandatory                           |
+| --------------------------- | -------------------------------------------------------------------------------------- | ----------------------------------- |
+| IOTEDGE_APIVERSION          | Version of IoT Edge; also used to determine if LNS is running as an IoT Edge module    | Yes, if running as IoT Edge module  |
+| IOTEDGE_IOTHUBHOSTNAME      | Host name of the IoT Hub                                                               | Yes                                 |
+| IOTEDGE_GATEWAYHOSTNAME     | Gateway host name                                                                      | Yes, if edgeHub gateway is enabled  |
```

Comment:
I would say that, unless someone is debugging locally, we can remove the IOTEDGE_ variables from this table as these are injected by IoT Edge runtime itself.